app/scripts/controllers/home.js

```diff
@@ -176,6 +176,7 @@ angular.module('noteeApp').controller('textInstanceCtrl', function ($scope, $uib
       }, function errorCallback(response) {
         console.log(response);
     });
+    location.reload();
```

Comment:
changed to $scope.loadData();

On Thu, Dec 17, 2015 at 1:08 AM, ananthkamal <notifications@github.com>
wrote:

> In app/scripts/controllers/home.js
> <https://github.com/elcinerkin/notee/pull/11#discussion_r47883002>:
>
> > @@ -176,6 +176,7 @@ angular.module('noteeApp').controller('textInstanceCtrl', function ($scope, $uib
> >        }, function errorCallback(response) {
> >          console.log(response);
> >      });
> > +    location.reload();
>
> Can you call $scope.loadData instead of location.reload() ? I think
> location.reload() will reload the whole page. After the modal is dismissed,
> call $scope.loadData() in HomeCtrl. Same applies to other type card modals
> too.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/elcinerkin/notee/pull/11/files#r47883002>.
>